rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -103,11 +112,21 @@ def run_service(self):
             endpoint=self.get_graph,
             methods=["GET"],
         )
+        router.add_api_route(
+            path="/graphs/{graph_id}/history",
+            endpoint=self.get_graph_history,
+            methods=["GET"],
+        )
         router.add_api_route(
             path="/graphs",
             endpoint=self.create_new_graph,
             methods=["POST"],
         )
+        router.add_api_route(
+            path="/graphs",
```

Comment:
Since this is an endpoint for updating a specific entity:
```suggestion
            path="/graphs/{graph_id}",
```